spec/factories/products.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :product do
+    breed "MyString"
+    dob "2017-06-19"
+    description "MyText"
+    price "MyString"
```

Comment:
Thanks for doing the factory for products! If we do switch price to a float, maybe we should change this to a float as well.